packages/twenty-front/src/modules/activities/types/Activity.ts

```diff
@@ -2,7 +2,7 @@ import { ActivityTarget } from '@/activities/types/ActivityTarget';
 import { Comment } from '@/activities/types/Comment';
 import { WorkspaceMember } from '@/workspace-member/types/WorkspaceMember';
 
-export type ActivityType = 'Task' | 'Note';
+export type ActivityType = 'TASK' | 'NOTE';
```

Comment:
nice :)

Reply:
We will need a command to update the existing data for cloud and self-hosting

Reply:
Done! Also tried to refactor a bit in my command to make things reusable, I feel we're not far from having tooling where writing these commands is very easy/fast.  

The new generic `upgrade-version` command works well.
For the `sharedBoilerplate` I created I'm not 100% sure so I didn't isolate it in a separate file yet, let's see when we have other commands to add, I feel it could help!